site/app/controllers/UserProfileController.php

```diff
@@ -143,11 +143,32 @@ public function changePronouns() {
         }
     }
 
+    /**
+     * @return JsonResponse
+     */
```

Comment:
If you're putting the return type in the function declaration I don't think you need it in the phpdoc as well
```suggestion
```

Reply:
@IDzyre thoughts?

Reply:
I believe if you use the `function(): JsonResponse {}` notation, you don't need to have the phpdoc declaration also, (unless it is an array and you want to specify the variables inside the array) but @williamjallen would know more.